### DISCUSSION TITLE
Is there a way to remove the title bar?

### DISCUSSION THREAD

**Discussion by thiagonunesbatista** [Kong/insomnia#3086] [Help] [answered]
I'm using Insomnia on Ubuntu 20.04. I 'm customizing my Insomnia, currentlty It looks like the image below

![Screenshot from 2021-02-13 11-49-51](https://user-images.githubusercontent.com/67487679/107852838-9a683a80-6df1-11eb-8a77-5556e8fff23b.png)

I'd like to remove this extra title bar where I have the Close Button, something similar to the custom title bar variant on vscode. 

![Screenshot from 2021-02-13 11-48-38](https://user-images.githubusercontent.com/67487679/107852790-6e4cb980-6df1-11eb-924c-6fe10f1336c2.png)

Is it possible? I couldn't find a way to do this just looking in the settings

**Comment by nijikokun** [comment, marked as answer]
Unfortunately not at the moment, we have thought about potentially making a *custom title bar*, but we haven't had any time to work on something like this yet.
